components/builder-web/habitat.conf.sample.js

```diff
@@ -22,6 +22,8 @@ habitatConfig({
     github_app_url: "https://github.com/apps/habitat-builder-dev",
     // The Habitat Builder GitHub app ID
     github_app_id: "5629",
+    // The GitHub redirect URI
```

Comment:
We should add a note here that this URI must exactly match what's defined in the GitHub app.